src/main/scala/com/gsoc/processor/AlertsProcessor.scala

```diff
@@ -0,0 +1,38 @@
+package com.gsoc.processor
+
+import java.nio.charset.StandardCharsets
+import java.nio.file.Paths
+
+import com.gsoc.csv.CsvParser
+import com.gsoc.gremlin_graph.GremlinGraph
+import com.gsoc.models.Model
+import gremlin.scala.ScalaGraph
+import zamblauskas.csv.parser.ColumnReads
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.Try
+
+final class AlertsProcessor(implicit ec: ExecutionContext) {
+
+  def startProcessor[T <: Model](graph: GremlinGraph[T])(implicit cr: ColumnReads[T]): Future[ScalaGraph] = {
+
+    for {
+
+      fileProcessor <- Future.fromTry {
+        Try {
+          scala.io.Source.fromURI(Paths.get("/Users/eusebiu/gsoc_2020_graphs/data/alerts.csv").toUri)(
```

Comment:
You should not hard-code path. It is much easier to make it configurable by users.